app/assets/javascripts/transitions/TransitionsPage.js

```diff
@@ -12,6 +12,7 @@ import GenericLoader from '../components/GenericLoader';
 import SectionHeading from '../components/SectionHeading';
 import School from '../components/School';
 import StudentPhotoCropped from '../components/StudentPhotoCropped';
+import HouseBadge from '../components/HouseBadge';
```

Comment:
@kevinrobinson I don't mean to step on anything here, but I think this slipped out and it's causing the js tests to fail